dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -22,9 +24,9 @@ ENV DEBIAN_FRONTEND=noninteractive
 # TODO: remove libpython3.6-dev, its and pip's dependencies if we can get pip3 directly
 # install subagent
 # clean up
-RUN apt-get update && apt-get install -y libperl5.20 libpci3 libwrap0 \
-            libexpat1-dev \
-            curl gcc && \
+RUN apt-get update && apt-get install -y --force-yes libperl5.20 libpci3 libwrap0 \
+            libexpat1-dev grub2-common bash-completion curl gcc \
```

Comment:
>grub2-common bash-completion [](start = 26, length = 28)

I think many dependencies are not needed if correctly refactored. If yes, please add a TODO comment.